Makefile

```diff
@@ -3,6 +3,8 @@ CONFIG_DIR = ./config
 BIN_DIR = ./node_modules/.bin
 BIN_FILE = $(DIST_DIR)/app.js
 
+recurse=$(wildcard $1$2) $(foreach d,$(wildcard $1*),$(call recurse,$d/,$2))
```

Comment:
Toutes les options de find ne sont pas portables effectivement. `type`et `name` semblent l'être d'après le man et des ressources sur le web comme http://www.in-ulm.de/~mascheck/various/find/.

Je trouve que la portabilité de findutils sort de ce qu'on pourrait définir comme un "périmètre de préoccupation" pour ce projet, ce sera plutot difficile de trouver un système ou tout tes scripts nodes fonctionneront sans erreurs, alors que cette implémentation très basique de find non.

A voir, c'est ici peut être une différence de point de vue mais je trouve que c'est mieux lorsque le code est lisible par les développeurs. Si il y a une erreur avec find, au moins, un néophyte comprendra. Avec ta définition de wildcard je suis pas sur.